pandas/core/internals/construction.py

```diff
@@ -536,9 +536,17 @@ def _list_of_series_to_arrays(data, columns, coerce_float=False, dtype=None):
 
 
 def _list_of_dict_to_arrays(data, columns, coerce_float=False, dtype=None):
+    """Convert list of OrderedDict to numpy array
+
+    if `columns` is not passed, column names are inferred from the records
```

Comment:
can you add a doc-string for the Parameters & Returns, types would be great too

Reply:
add Parameters & Returns, skipped types